axum/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
+- **added:** `axum_extra::middleware::from_fn` has been moved into the main axum crate. It is used
```

Comment:
For people who haven't seen it, "has been moved" might be confusing. I would suggest adding "from axum-extra" or just saying that it has been added.

Reply:
Good point. I've rephrased it a bit.